examples/Beehive2D.jl

```diff
@@ -566,7 +566,7 @@ ls(sfg, :x19)
 
 pts = approxConv(sfg, :x19l0f1, :x19)
 
-X19 = manikde!(pts, Pose2().manifolds)
+X19 = manikde!(pts, getManifolds(Pose2()))
```

Comment:
This should work `manikde!(pts, Pose2())`.

Reply:
Ok, I don't know the API yet. I just used direct replacement.  Should we change all of them?

Reply:
no need to change them, just wanted to share so that you are aware.  The internal function are doing the same thing you are `getManifolds(T())`